### ISSUE TITLE
Echo-cancelation doesn't work on Android

### ISSUE THREAD

**Issue by Michael1972** [BearWare/TeamTalk5#1083]
Hi, on android-devices the echo-canceler doesn't seem to work. While it's possible to use latest Teamtalk on IOS, mac or windows with enabled echo-canceling, having no problems even without a headset, Teamtalk for android produces echos, even if the echo-cancelation is enabled. Currently, the only work-around is, always turning the microphone on and off. Could you please fix this? 
Thank you.

**Comment by Oreonan** [comment]
For me, sound effects like echo cancellation on mobile clients are a problem. @bear101 it would not be possible to use WebRTC also on mobile clients  instead of native phone features that are more or less good?

**Comment by bear101** [comment]
I mainly test with iOS and there the echo cancellation works really well

**Comment by Michael1972** [comment]
@Bear101: You are right. I have an IOS as well as an android-device. On IOS echo-cancelation works perfectly. On android it obviously doesn't work at all.
